### PR TITLE
Make it easier to test prow locally.

### PR DESCRIPTION
**Pull request by spxtr**
Individual commits are easier to follow. The readme change explains the purpose.

The +lines is misleading since I add a couple example webhook payloads that are pretty lengthy.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1197)

### CODE REVIEW

File: prow/Makefile [hunk 1]
```diff
@@ -39,18 +39,6 @@ JENKINS_ADDRESS_FILE = ${HOME}/jenkins-address
 # Service account key for bootstrap jobs.
 SERVICE_ACCOUNT_FILE = ${HOME}/service-account.json
 
-# Useful rules:
```
Comment (rmmh) [inline prow/Makefile]: why delete this? it seems helpful

File: prow/cmd/phony/examples/lgtm_comment.json [hunk 2]
```diff
@@ -0,0 +1,258 @@
+{
```
Comment (rmmh) [inline prow/cmd/phony/examples/lgtm_comment.json]: consider stripping the _url components from this file for brevity

Comment (spxtr) [inline prow/cmd/phony/examples/lgtm_comment.json]: I use `html_url` in a few places.

**Comment by spxtr** [comment]
@rmmh, @foxish ping. I'm going to make some stability changes soon and I'd like this to be in place.

**Comment by foxish** [comment]
LGTM from me as well. This is a great change and will help test this much better. Thanks!
Would be nice if you could explain further the purpose of hook, line and sinker (because they are non-obvious), in the readme.

**Comment by spxtr** [comment]
Will do in a future PR where I improve lots of docs, probably later this week.

**Pull request merged**